ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-msdp.md

```diff
@@ -0,0 +1,145 @@
+# router-msdp
+# Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Authentication](#authentication)
+- [Monitoring](#monitoring)
+- [Internal VLAN Allocation Policy](#internal-vlan-allocation-policy)
+  - [Internal VLAN Allocation Policy Summary](#internal-vlan-allocation-policy-summary)
+- [Interfaces](#interfaces)
+- [Routing](#routing)
+  - [IP Routing](#ip-routing)
+  - [IPv6 Routing](#ipv6-routing)
+- [Multicast](#multicast)
+  - [Router MSDP](#router-msdp)
+- [Filters](#filters)
+- [ACL](#acl)
+- [Quality Of Service](#quality-of-service)
+
+# Management
+
+## Management Interfaces
+
+### Management Interfaces Summary
+
+#### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+#### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+# Authentication
+
+# Monitoring
+
+# Internal VLAN Allocation Policy
+
+## Internal VLAN Allocation Policy Summary
+
+**Default Allocation Policy**
+
+| Policy Allocation | Range Beginning | Range Ending |
+| ------------------| --------------- | ------------ |
+| ascending | 1006 | 4094 |
+
+# Interfaces
+
+# Routing
+
+## IP Routing
+
+### IP Routing Summary
+
+| VRF | Routing Enabled |
+| --- | --------------- |
+| default | False |
+
+### IP Routing Device Configuration
+
+```eos
+```
+## IPv6 Routing
+
+### IPv6 Routing Summary
+
+| VRF | Routing Enabled |
+| --- | --------------- |
+| default | False |
+
+# Multicast
+
+## Router MSDP
+
+### Router MSDP Peers
+
+| Peer Address | VRF | Mesh Groups | Local Interface | Description |
+| ------------ | --- | ----------- | --------------- | ----------- |
+| 1.2.3.4 | default | MG1, MG2 | Loopback11 | Some kind of MSDP Peer |
+| 4.3.2.1 | default | - | Loopback21 | - |
+| 2.3.4.5 | RED | - | Loopback13 | Some other kind of MSDP Peer |
+
+### Router MSDP Device Configuration
+
+```eos
+!
+router msdp
+   group-limit 100 source 10.0.1.0/24
+   group-limit 123 source 10.0.123.0/24
+   originator-id local-interface Loopback10
+   rejected-limit 123
+   forward register-packets
+   connection retry interval 5
+   !
+   peer 1.2.3.4
+      default-peer prefix-list PLIST1
```

Comment:
Should the information about default-peer and any associated prefix-list not be included in the overview table for easy readability?

Reply:
Added, thanks!